### PR TITLE
[IPCONFIG] Update Polish (pl-PL) translation

### PR DESCRIPTION
**Pull request by pithwz**
ipconfig [pl-pl] lang update

### CODE REVIEW

File: base/applications/network/ipconfig/lang/pl-PL.rc [hunk 1]
```diff
@@ -5,79 +5,80 @@
...
+    IDS_NODETYPEBCAST       "\tTyp węzła . . . . . . . . . . . . : Transmisja\n"
+    IDS_NODETYPEP2P         "\tTyp węzła . . . . . . . . . . . . : Peer To Peer\n"
+    IDS_NODETYPEMIXED       "\tTyp węzła . . . . . . . . . . . . : Mieszany\n"
+    IDS_NODETYPEHYBRID      "\tTyp węzła . . . . . . . . . . . . : Hybrydowy\n"
+    IDS_NODETYPEUNKNOWN     "\tTyp węzła . . . . . . . . . . . . : nieznany\n"
+    IDS_IPROUTINGNO         "\tRouting IP włączony . . . . . . . . : Nie\n"
+    IDS_IPROUTINGYES        "\tRouting IP włączony . . . . . . . . : Tak\n"
+    IDS_WINSPROXYNO         "\tSerwer WINS Proxy włączony . . . . . . . . : Nie\n"
+    IDS_WINSPROXYYES        "\tSerwer WINS Proxy włączony . . . . . . . . : Tak\n"
+    IDS_DNSSUFFIXLIST       "\tLista przeszukiwania sufiksów DNS . . . . . . : %hs\n"
```
Comment (binarymaster) [inline base/applications/network/ipconfig/lang/pl-PL.rc:27]:
```suggestion
    IDS_HOSTNAME            "\tNazwa hosta . . . . . . . . . . . : %hs\n"
    IDS_PRIMARYDNSSUFFIX    "\tSufiks podstawowej domeny DNS . . : %hs\n"
    IDS_NODETYPEBCAST       "\tTyp węzła . . . . . . . . . . . . : Transmisja\n"
    IDS_NODETYPEP2P         "\tTyp węzła . . . . . . . . . . . . : Peer To Peer\n"
    IDS_NODETYPEMIXED       "\tTyp węzła . . . . . . . . . . . . : Mieszany\n"
    IDS_NODETYPEHYBRID      "\tTyp węzła . . . . . . . . . . . . : Hybrydowy\n"
    IDS_NODETYPEUNKNOWN     "\tTyp węzła . . . . . . . . . . . . : nieznany\n"
    IDS_IPROUTINGNO         "\tRouting IP włączony . . . . . . . : Nie\n"
    IDS_IPROUTINGYES        "\tRouting IP włączony . . . . . . . : Tak\n"
    IDS_WINSPROXYNO         "\tSerwer WINS Proxy włączony. . . . : Nie\n"
    IDS_WINSPROXYYES        "\tSerwer WINS Proxy włączony. . . . : Tak\n"
    IDS_DNSSUFFIXLIST       "\tLista przeszukiwania sufiksów DNS : %hs\n"
```

File: base/applications/network/ipconfig/lang/pl-PL.rc [hunk 2]
```diff
@@ -5,79 +5,80 @@
...
+    IDS_AUTOCONFIGYES       "\tAutokonfiguracja włączona . . . . : Tak\n"
+    IDS_IPADDRESS           "\tAdres IP. . . . . . . . . . . . : %hs\n"
+    IDS_SUBNETMASK          "\tMaska podsieci . . . . . . . . . . . : %hs\n"
+    IDS_DEFAULTGATEWAY      "\tBrama domyślna . . . . . . . . . : %hs\n"
+    IDS_DHCPSERVER          "\tSerwer DHCP . . . . . . . . . . . : %hs\n"
+    IDS_DNSSERVERS          "\tSerwer DNS . . . . . . . . . . . : %hs\n"
+    IDS_PRIMARYWINSSERVER   "\tPodstawowy serwer WINS . . . . . . . : %hs\n"
+    IDS_SECONDARYWINSSERVER "\tPomocniczy serwer WINS . . . . . . : %hs\n"
+    IDS_LEASEOBTAINED       "\tDzierżawa uzyskana . . . . . . . . . . : %s\n"
+    IDS_LEASEEXPIRES        "\tDzierżawa wygasa . . . . . . . . . . : %s\n"
```
Comment (binarymaster) [inline base/applications/network/ipconfig/lang/pl-PL.rc:45]:
```suggestion
    IDS_MEDIADISCONNECTED   "\tStan nośnika. . . . . . . . . . . : Nośnik odłączony\n"
    IDS_CONNECTIONDNSSUFFIX "\tSufiks DNS konkretnego połączenia : %hs\n"
    IDS_DESCRIPTION         "\tOpis. . . . . . . . . . . . . . . : "
    IDS_PHYSICALADDRESS     "\tAdres fizyczny. . . . . . . . . . : %hs\n"
    IDS_DHCPNO              "\tDHCP włączone . . . . . . . . . . : Nie\n"
    IDS_DHCPYES             "\tDHCP włączone . . . . . . . . . . : Tak\n"
    IDS_AUTOCONFIGNO        "\tAutokonfiguracja włączona . . . . : Nie\n"
    IDS_AUTOCONFIGYES       "\tAutokonfiguracja włączona . . . . : Tak\n"
    IDS_IPADDRESS           "\tAdres IP. . . . . . . . . . . . . : %hs\n"
    IDS_SUBNETMASK          "\tMaska podsieci. . . . . . . . . . : %hs\n"
    IDS_DEFAULTGATEWAY      "\tBrama domyślna. . . . . . . . . . : %hs\n"
    IDS_DHCPSERVER          "\tSerwer DHCP . . . . . . . . . . . : %hs\n"
    IDS_DNSSERVERS          "\tSerwer DNS. . . . . . . . . . . . : %hs\n"
    IDS_PRIMARYWINSSERVER   "\tPodstawowy serwer WINS. . . . . . : %hs\n"
    IDS_SECONDARYWINSSERVER "\tPomocniczy serwer WINS. . . . . . : %hs\n"
    IDS_LEASEOBTAINED       "\tDzierżawa uzyskana. . . . . . . . : %s\n"
    IDS_LEASEEXPIRES        "\tDzierżawa wygasa. . . . . . . . . : %s\n"
```

File: base/applications/network/ipconfig/lang/pl-PL.rc [hunk 3]
```diff
@@ -5,79 +5,80 @@
...
+    IDS_DNSSECTIONADDITIONAL "\tSekcja . . . . . . . : Dodatkowa\n"
+    IDS_DNSTYPEA             "\tRekord (hosta) . . . : %ls\n"
+    IDS_DNSTYPENS            "\tRekord NS . . . . . . : %ls\n"
+    IDS_DNSTYPECNAME         "\tRekord CNAME . . . . . : %ls\n"
+    IDS_DNSTYPESOA           "\tRekord SOA . . . . . . : \n"
+    IDS_DNSTYPEPTR           "\tRekord PTR . . . . . . : %ls\n"
+    IDS_DNSTYPEMX            "\tRekord MX . . . . . . : \n"
+    IDS_DNSTYPEAAAA          "\tRekord AAAA . . . . . : %ls\n"
+    IDS_DNSTYPEATMA          "\tRekord ATMA . . . . . : \n"
+    IDS_DNSTYPESRV           "\tRekord SRV. . . . . . : \n"
```
Comment (binarymaster) [inline base/applications/network/ipconfig/lang/pl-PL.rc:81]:
```suggestion
    IDS_DNSRECORDNAME        "\tNazwa rekordu . . . . : %ls\n"
    IDS_DNSRECORDTYPE        "\tTyp rekordu . . . . . : %hu\n"
    IDS_DNSRECORDTTL         "\tCzas wygaśnięcia. . . : %lu\n"
    IDS_DNSRECORDLENGTH      "\tDługość danych. . . . : %hu\n"
    IDS_DNSSECTIONQUESTION   "\tSekcja. . . . . . . . : Pytanie\n"
    IDS_DNSSECTIONANSWER     "\tSekcja. . . . . . . . : Odpowiedź\n"
    IDS_DNSSECTIONAUTHORITY  "\tSekcja. . . . . . . . : Źródło\n"
    IDS_DNSSECTIONADDITIONAL "\tSekcja. . . . . . . . : Dodatkowa\n"
    IDS_DNSTYPEA             "\tRekord (hosta) A. . . : %ls\n"
    IDS_DNSTYPENS            "\tRekord NS . . . . . . : %ls\n"
    IDS_DNSTYPECNAME         "\tRekord CNAME. . . . . : %ls\n"
    IDS_DNSTYPESOA           "\tRekord SOA. . . . . . : \n"
    IDS_DNSTYPEPTR           "\tRekord PTR. . . . . . : %ls\n"
    IDS_DNSTYPEMX            "\tRekord MX . . . . . . : \n"
    IDS_DNSTYPEAAAA          "\tRekord AAAA . . . . . : %ls\n"
    IDS_DNSTYPEATMA          "\tRekord ATMA . . . . . : \n"
    IDS_DNSTYPESRV           "\tRekord SRV. . . . . . : \n"
```

**Review by binarymaster — COMMENTED** [review]
Alignment corrections

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

**Review by Saibamen — APPROVED** [review]

**Review by archeYR — APPROVED** [review]

**Pull request merged**